packages/openneuro-app/src/scripts/refactor_2021/search/search-container.tsx

```diff
@@ -63,7 +70,25 @@ const SearchContainer: FC = () => {
           <StudyDomainSelect />
         </>
       )}
-      renderSearchResultsList={() => null}
+      renderSearchResultsList={() =>
+        loading ? (
```

Comment:
can you also wrap a condition around the loading condition that wont render the block at all if `results === 0`